packages/datatrak-web/package.json

```diff
@@ -55,7 +55,8 @@
     "start-web-config-server": "yarn workspace @tupaia/web-config-server start-dev",
     "start-entity-server": "yarn workspace @tupaia/entity-server start-dev",
     "start-servers": "npm-run-all -c -l -p start-central-server start-web-config-server start-datatrak-web-server start-entity-server",
-    "test": "yarn package:test"
+    "test": "yarn test:jest",
+    "test:jest": "yarn package:test --env=jsdom"
```

Comment:
Any reason we need both of these?

Reply:
I just copied what was in the package.json for psss. I think that it is a convention because I've seen it in some npm modules around the place. Maybe so we organise scripts for other types of test such as cypress